lms/djangoapps/courseware/features/homepage.feature

```diff
@@ -1,12 +1,12 @@
 @shard_1
 Feature: LMS.Homepage for web users
   In order to get an idea what edX is about
-  As a an anonymous web user
+  As an anonymous web user
   I want to check the information on the home page
 
-  Scenario: User can see the "Login" button
+  Scenario: User can see the "Sign in" button
```

Comment:
@griffresch instances of "Log in" have been changed to "Sign in." The issue is that we also use the phrase "Log out." This leaves us with "Sign in" and "Log out," hence the inconsistency.